.make.defaults

```diff
@@ -306,7 +289,7 @@ __check_defined = \
 	if [ -e ../python ]; then								\
```

Comment:
And again,

`$(call check_defined, DOCKER_LOCAL_IMAGE)`

Reply:
I need to keep changes to the Makefile to a minimum as most of it will change in a second iteration once I get a good handle on it. If this is not causing a bug right now or breaking the code, I won't be changing it in this iteration.